tests/.suitespec.json

```diff
@@ -116,6 +116,7 @@
         "flask": [
             "ddtrace/contrib/flask/*",
             "ddtrace/contrib/flask_cache/*"
+            "ddtrace/contrib/flask_login/*"
```

Comment:
```suggestion
            "ddtrace/contrib/flask_login/*",
```

Reply:
Sorry, I apparently don't know how to make a single comment with a suggestion that spans multiple lines.